### ISSUE TITLE
Missing "tslib" leads to error (v0.8.0)

### ISSUE THREAD

**Issue by yperson** [Mattihew/k6-to-junit#1]
Hi,

it seems like version 0.8.0 introduced an issue when running k6-to-junit after installing it via npm. The tslib is not installed by default and therefore leads to an error preventing k6-to-junit from properly running.

I guess it should be added to the install process?

```
internal/modules/cjs/loader.js:613
    throw err;
    ^

Error: Cannot find module 'tslib'
Require stack:
- /usr/local/lib/node_modules/k6-to-junit/dist/command.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:610:15)
    at Function.Module._load (internal/modules/cjs/loader.js:526:27)
    at Module.require (internal/modules/cjs/loader.js:666:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/usr/local/lib/node_modules/k6-to-junit/dist/command.js:4:17)
    at Module._compile (internal/modules/cjs/loader.js:759:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
    at Module.load (internal/modules/cjs/loader.js:628:32)
    at Function.Module._load (internal/modules/cjs/loader.js:555:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:824:10)
write /dev/stdout: broken pipe
```

**Comment by Mattihew** [comment]
Thanks, this is now fixed in 0.8.1

**Comment by yperson** [comment]
Thanks, awesome!
